### DISCUSSION TITLE
[Suggestion] Latest transactions suggestions

### DISCUSSION THREAD

**Discussion by JannemanDev** [iotaledger/firefly#1559] [Features / Enhancements]
-Show negative/outgoing transactions red and positive/incoming as green.
Since you already using arrows at the left, make them red or green. And/or you could also make the amount red or green. Btw I don't find the up/down arrows very intuitive. Up arrow communicates to me that my balance would go up. Also my first instinct reaction was that it would give me a dropdown with extra detailed transaction info. Better is to remove the arrow and just color the amount red or green.
-Also show a little description if available. Check out the financial app of your local bank and take all the good stuff. They have years of a head start (don't get me wrong Firefly already looks great).

![image](https://user-images.githubusercontent.com/13236774/112294199-22fcb500-8c93-11eb-96b6-6ef46870886e.png)

**Comment by JannemanDev** [comment]
Other considerations regarding this issue see https://github.com/iotaledger/firefly/issues/664

**Comment by JannemanDev** [comment]
Update : I noticed the icon before transaction has a third variant: internal transaction, see first transaction in the updated picture above.
So I guess we keep the icons but change the up and down into a box (representing a wallet) with a red outgoing arrow (send funds) and a box with a green incoming arrow (incoming funds).

**Comment by dev0pa** [comment]
![1](https://user-images.githubusercontent.com/81360970/112462284-2c545300-8d61-11eb-972e-045b0eb2783c.jpg)

**Comment by Travetown** [comment]
Perhaps for internal transactions it would be better not to show + or - at all (and then leave it white). 
In the "Last Transactions" overview, only the transaction for the account that leads the ranking is displayed.
Or both entries should be displayed (+ and -)
